frontends/election-manager/src/components/smartcard_modal/card_details_view.tsx

```diff
@@ -1,5 +1,16 @@
 import React from 'react';
+import { Button } from '@votingworks/ui';
+import { CardProgramming } from '@votingworks/types';
 
-export function CardDetailsView(): JSX.Element {
-  return <h2>Card Details</h2>;
+interface Props {
+  card: CardProgramming;
+}
+
+export function CardDetailsView({ card }: Props): JSX.Element {
```

Comment:
I set up these placeholder UIs to 1) demonstrate usage of the `CardProgramming` interface and 2) allow me to test manually. I'll be building out the real UIs in subsequent PRs, which is why this PR doesn't include tests at the VxAdmin app level, yet